zebra-network/src/peer_set/set.rs

```diff
@@ -170,14 +170,23 @@ where
     D::Error: Into<BoxError>,
     C: ChainTip,
 {
+    // Peer Tracking: New Peers
+    //
```

Comment:
```suggestion
    // Peer Tracking: New Peers

```
I am not sure if this is better but i think it can be a bit confusing to realize about the themes as it is now. Id this is changed then it should be done for all the other section headers as well.

Reply:
Rustfmt will remove the empty line if we do this change 😅

Reply:
Sorry, I should have tried it.